TestCases/py_wrapper/rotating_cylinder/run.py

```diff
@@ -0,0 +1,26 @@
+import pysu2			            # imports the SU2 wrapped module
```

Comment:
Thanks :+1: to make a regression test from this you need to add it to parallel_regression.py

Reply:
Add our license header to this file please (since it's also code).